lib/class-wp-rest-dependencies-controller.php

```diff
@@ -191,11 +196,16 @@ public function get_items_permissions_check( $request ) {
 	 * @return bool|true|WP_Error
 	 */
 	public function get_item_permissions_check( $request ) {
-		if ( $this->check_handle( $request['handle'] ) ) {
+		$check = $this->check_read_permission( $request['handle'] );
+		if ( is_wp_error( $check ) ) {
+			return $check;
+		} else if ( true === $check ) {
+			return $check;
+		} else if (  current_user_can( 'manage_options' ) ) {
 			return true;
```

Comment:
Same as above, this is not an error just a note, there is no need for `else/if` when we `return`. Perhaps change the code as per the example posted above?